easybuild/easyconfigs/l/libcircle/libcircle-0.3-gompi-2020a-rc3.eb

```diff
@@ -0,0 +1,41 @@
+#    With <3 for EasyBuild
+#
+#    EasyConfig for libcircle:
+#    ----------------------------------------------------------------------------
+#    Copyright: 2014 - Gregor Mendel Institute of Molecular Plant Biology GmBH
+#    License: MIT
+#    Authors: Petar <petar.forai@gmi.oeaw.ac.at> Forai
+#    ----------------------------------------------------------------------------
+
+easyblock = 'ConfigureMake'
+
+name = 'libcircle'
+version = '0.3'
+
+homepage = 'https://github.com/hpc/libcircle/'
+
+description = """
+ An API to provide an efficient distributed queue on a cluster. libcircle is an
+ API for distributing embarrassingly parallel workloads using self-stabilization.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2020a-rc3'}
```

Comment:
`gompi/2020a` should be final now in the `2020a` branch, so please sync and update this to `'2020a'`?